packages/gatsby-theme/src/blocks/ShareBlock/index.js

```diff
@@ -3,23 +3,19 @@ import PropTypes from "prop-types"
 import { Box } from "theme-ui"
 import { Share } from "ui"
 
-const ShareBlock = ({ heading, data, ...props }) => {
+const ShareBlock = ({ data, ...props }) => {
   return (
     <Box variant='wrapper' __themeKey='container'>
```

Comment:
Just in case, we should accept an sx prop
```suggestion
const ShareBlock = ({ data, sx, ...props }) => {
  return (
    <Box variant='wrapper' __themeKey='container' sx={sx}>
```